internal/field.go

```diff
@@ -104,7 +104,8 @@ func toCamelInitCase(name string, initUpper bool) string {
 			continue
 		}
 		if p == "id" {
-			out += "ID"
+			// Smartpass structs use Id instead of the default ID
+			out += "Id"
```

Comment:
This is unrelated to the other changes, but we had started manually configuring sqlc to rewrite `SchoolID` to `SchoolId`, etc. This changes the behavior at the source. (Note:  I could remove the `if` entirely, but it seemed clearer to keep it if we need to merge upstream changes.)

Reply:
Ah sweet. Thank you.

Reply:
This didn't fully work, but I'll hold off on the rest of the changes for now. I see what needs to be done, it'll just be easier to review separately.